documentation/modules/upgrading/con-upgrade-sequence.adoc

```diff
@@ -14,6 +14,10 @@ Strimzi {ProductVersion} requires Kubernetes {KubernetesVersion}.
 +
 You can xref:con-upgrade-cluster-{context}[upgrade Kubernetes with minimal downtime].
 
-. xref:assembly-upgrade-cluster-operator-{context}[Upgrade the Cluster Operator].
+. xref:assembly-upgrade-{context}[Upgrade the Cluster Operator].
 
-. xref:assembly-upgrading-kafka-versions-{context}[Upgrade all Kafka brokers and client applications] to the latest supported Kafka version.
\ No newline at end of file
+. Upgrade Kafka depending on the cluster configuration:
+.. If using Kafka in KRaft mode, use the latest supported Kafka version and `spec.kafka.metadataVersion` to xref:proc-upgrade-kafka-kraft-{context}[upgrade all Kafka brokers and client applications].
+.. If using Kafka with ZooKeeper for cluster management, use the latest supported Kafka version and `inter.broker.protocol.version` to xref:assembly-upgrade-zookeeper-{context}[upgrade all Kafka brokers and client applications].
```

Comment:
I'm not sure I understand the meaning of the `use the latest supported` in both of these bullets. The point of upgrade is to upgrade to a newer version. Should it be something like `in order to use the latest supported` or something like that?

Reply:
Supported by Strimzi. I changed to "update the..."